Makefile

```diff
@@ -11,6 +11,7 @@ include build/scripts/testing.mk
 include contracts/Makefile 
```

Comment:
Currently the pre-requisites steps are listed in the `fullnode.mk` file. To mount these files such as config.toml and genesis.json, building the binary won't help.
So thinking to spin up beacond using docker containers to be able to mount volumes.